rpc/ut/link_ut.c

```diff
@@ -334,7 +335,7 @@ static struct m0_fop *ut_fop_alloc(const char *name, uint32_t cmd)
 	M0_ALLOC_PTR(rfop);
 	M0_UT_ASSERT(rfop != NULL);
 
-	m0_buf_init(&rfop->ss_name, (void *)name, strlen(name));
+	m0_buf_init(&rfop->ss_name, (void *)name, strnlen(name, CONF_SERVICE_IOS_LEN));
```

Comment:
You can Ignore instead of fixing it as it is a constant string.

Reply:
Done.